goss-testing/scripts/python/check_for_unused_drives.py

```diff
@@ -76,17 +76,20 @@
 hw_type_map = {
     "GIGA-BYTE TECHNOLOGY CO., LTD": "gigabyte",
     "Hewlett Packard Enterprise": "hpe",
-    "Intel Corporation": "intel" }
+    "Intel Corporation": "intel",
+    "vshasta": "vshasta" }
```

Comment:
I don't see any other place which would ever set the hardware type to vshasta. Don't you need to modify that code as well?

Also, if this is done, isn't the vshasta variable then redundant? The hardware type could just be checked instead, it seems like, unless I'm missing something here.

Reply:
There actually is no ipmitool on vshasta.  We just run a mock version of it, so we just fake it.

https://github.com/Cray-HPE/livecd-gcp-infrastructure/pull/174/files#diff-34e0270d5867cf8e17dc52ef091290e1cf3b721c71c3d9181004129abf8aa7b5R207

Reply:
I think we may still want this code.  I think this test should still run on vshasta, rather then being skipped on the `is_vshasta` conditional.

Reply:
Yeah, I agree. I was saying that if you have the hardware manufacturer variable set to `vshasta`, then you don't also need the other `vshasta` boolean variable. You could just check the manufacturer variable. But it's not a big deal.

Reply:
True.  It is redundant, but it is a very small component of vshasta and mostly superfluous because we just use it to fake out some other scripts that depend on it, but we actually have the data elsewhere